src/components/Vulnurabilities/CcviThermometer/CcviThermometer.tsx

```diff
@@ -19,8 +24,23 @@ const CcviThermometer: React.FC<{ overallScore: number }> = ({
 
   const pointerX = scaleWidth(overallScore);
 
+  const level = getCcviLevel(overallScore);
+
+  // todo (chelsi): add location name, possibly edit copy
+  const title = level
+    ? `Thermometer image showing that locationName's vulnurability level is ${getCcviLevelName(
+        level,
+      ).toLowerCase()}.`
+    : 'Thermometer image showing locationName vulnurability level.';
```

Comment:
maybe remove the `locationName` from the copy until we can pass it as parameter?

Reply:
Also there is a typo "vulnurability" -> "vulnerability"

Reply:
ahh